examples/index.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<title>term.js</title>
+<!--
+  term.js
+  Copyright (c) 2012-2013, Christopher Jeffrey (MIT License)
+-->
+<style>
+  html {
+    background: #555;
+  }
+
+  h1 {
+    margin-bottom: 20px;
+    font: 20px/1.5 sans-serif;
+  }
+
+/*
+  .terminal {
+    float: left;
+    border: #000 solid 5px;
+    font-family: "DejaVu Sans Mono", "Liberation Mono", monospace;
+    font-size: 11px;
+    color: #f0f0f0;
+    background: #000;
+  }
+
+  .terminal-cursor {
+    color: #000;
+    background: #f0f0f0;
+  }
+*/
+</style>
+<h1>term.js</h1>
+<script src="term.js"></script>
+<script>
+;(function() {
+  window.onload = function() {
+    var socket = new WebSocket('ws://localhost:8081/a');
+    socket.onopen = function() {
+      var term = new Terminal({
+        cols: 80,
+        rows: 24,
+        useStyle: true,
+        screenKeys: true,
+        cursorBlink: false
+      });
+
+      term.on('data', function(data) {
+        socket.send(data);
```

Comment:
Don't you need like a message `code` ? to keep a protocol we can extend... Also make sure we send the data binary.. Probably `term.on('data',...` gives you a string right... So probably utf-8 encode it... browserify is our friend here because then we can use `new Buffer(, 'utf8')` in the browser. Okay, I'm not sure about `utf-8` we should check how data is encoded.

Anyways, let's try to build a generic client side library, that will do the decoding of message codes, encoding of message codes, etc...